### PR TITLE
Bug 1251988 - update wptview link string

### PR DESCRIPTION
**Pull request by martiansideofthemoon**
@wlach , @jgraham is this alright?

---

This change is [Reviewable](https://reviewable.io/reviews/mozilla/treeherder/1345)

### CODE REVIEW

File: ui/plugins/job_details/main.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
         <a ng-switch-when="link" title="{{line.value}}"
            href="{{line.url}}" target="_blank">{{line.value}}</a>
         <span ng-if="line.value.endsWith('raw.log')">
-          - <a title="{{line.value}}" href="http://mozilla.github.io/wptview/#/?urls={{line.url | encodeURIComponent}},{{job_details[buildernameIndex].value | encodeURIComponent}}">open in wptview</a>
+          - <a title="{{line.value}}" href="http://mozilla.github.io/wptview/#/?urls={{line.url | encodeURIComponent}},{{job_details[buildernameIndex].value | encodeURIComponent}}" target="results_viewer">open in results viewer</a>
```
Comment (wlach) [inline ui/plugins/job_details/main.html]: Let's say "test results viewer" instead of just "results viewer"

**Comment by edmorley** [comment]
(Note to self: this has been reviewed, comments in https://bugzilla.mozilla.org/show_bug.cgi?id=1251988#c17)

**Comment by edmorley** [comment]
Is this change still wanted? (After the review comments addressed)
Or should we close out this PR for now? :-)

**Comment by martiansideofthemoon** [comment]
@edmorley We want to open the link in a new tab meant for wptview and refresh the tab every time the link is pressed. Right @jgraham ?

**Comment by edmorley** [comment]
@jgraham ping?

**Comment by jgraham** [comment]
Yeah, I think that's right. The patch looks correct to me; does it not work?

**Comment by edmorley** [comment]
@martiansideofthemoon, fancy rebasing this PR? :-)

**Comment by martiansideofthemoon** [comment]
I don't think `wptview` links work anymore (atleast when I last checked) due to changes in the HTTP headers I believe. I think this can be safely closed @jgraham ?

**Comment by edmorley** [comment]
> I don't think wptview links work anymore (atleast when I last checked) due to changes in the HTTP headers I believe. I think this can be safely closed @jgraham ?

In the absence of a reply, closing.

**Pull request closed without merging**